### ISSUE TITLE
set fabric8.skip.build.pom to true by default?

### ISSUE THREAD

**Issue by Ladicek** [openshiftio/booster-parent#22]
With the upgrade of Fabric8 Maven plugin to version 3.5.35, should we maybe set `fabric8.skip.build.pom` to `true` by default in the parent POM? It works around the multi-module build problem, per https://github.com/fabric8io/fabric8-maven-plugin/issues/1184#issuecomment-368308674 (and below).

This would allow boosters to adopt the latest booster parent and hence move to OpenShift 3.7.

CC @metacosm.

**Comment by metacosm** [comment]
That's what I did in my local test in the Spring Boot specific parent. Here's the appropriate change (based on documentation interpolation since the documentation actually doesn't say anything about this specific property):

```xml
<configuration>
  <skipBuildPom>true</skipBuildPom>
</configuration>
```

But, yes, if we commit to FMP 3.5.35 and the workaround, this change should be made in this project for all runtimes to benefit from.

**Comment by Ladicek** [comment]
Yea, or we can add it in `<properties>`, should be the same, I don't mind :-)

**Comment by maxandersen** [comment]
according to https://github.com/fabric8io/fabric8-maven-plugin/issues/1184#issuecomment-368517072 the proper fix is to put the dependency of fmp into <pluginManagement> - would that be an acceptable change ?

**Comment by Ladicek** [comment]
Actually this parent POM _does_ define FMP in `<pluginManagement>`, it's the multi-module boosters that don't.

**Comment by Ladicek** [comment]
Thanks!

**Comment by metacosm** [comment]
It looks that this workaround is not needed anymore.

**Comment by Ladicek** [comment]
That's probably right, I think the FMP behavior was reverted to the previous state, to avoid behavioral regression.

**Comment by metacosm** [comment]
I checked that there was no issue with a multi-module booster with FMP `3.5.40` without that property set.

**Comment by metacosm** [comment]
On the other hand, the new behavior will be reactivated when a new minor version of FMP is released (I assume `3.6`?), though, I still don't agree with the rationale given to justify that new behavior… :/

**Comment by Ladicek** [comment]
Yes, I think there was a proposal to activate the new behavior by default in a new major version. Not sure if that would mean 3.6 or 4.0.
